### PR TITLE
Allow querying for active games

### LINKED ISSUE #19: Ability to get list of actives games for user
As a client app
I can retrieve a list of active games for the current user (including a flag indicating whether I can play my turn)
So that I can allow the client to to pick which game they want to play

(copy/paste from the Google drive document):

GET /api/games?userName=FungusMungus&active=1

Get a list of active games for the specified user

Example JSON response:
```
{
   "Games":[
      {
         "id":2391,
         "numberOfHumanPlayers":2,
         "numberOfAiPlayers":1,
         "dateCreated":"2019-02-07",
         "nextRoundAvailable":true
      },
      {
         "id":2392,
         "numberOfHumanPlayers":4,
         "numberOfAiPlayers":0,
         "dateCreated":"2019-02-08",
         "nextRoundAvailable":false
      }
   ]
}
```

### PR DESCRIPTION
**Pull request by Serneum**
Some assumptions I made:
- A game is only active if it is in the `Started` state
- Both `true` and `1` should be capable of returning games in the `Started` state
- If no `active` param is passed in, or the value is `false` (or `0`), return all games regardless of state 
- Any other value for `active` should return a `"Bad Request"`

Closes #19

### CODE REVIEW

**Comment by Serneum** [comment]
I still need to add the `nextRoundAvailable` stuff. Trying to think through how to do that

File: test/fungus_toast/games/games_test.exs [hunk 1]
```diff
@@ -3,8 +3,9 @@ defmodule FungusToast.GamesTest do
 
   alias FungusToast.Accounts
   alias FungusToast.Games
+  alias FungusToast.Players
 
-  def user_fixture(attrs \\ %{}) do
+  defp user_fixture(attrs \\ %{}) do
     {:ok, user} =
       attrs
       |> Enum.into(%{user_name: "testUser"})
```
Comment (daveshah) [inline test/fungus_toast/games/games_test.exs]: Is this being used? Just from the REPL, `Enum.into` doesn't return `{:ok, ...}` or `{:error, ...}`

Comment (Serneum) [reply test/fungus_toast/games/games_test.exs]: Accidentally resolved this. The Enum.into feeds into an Accounts.create that doesn't seem to be in the diff

File: FungusToast.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+	"info": {
+		"_postman_id": "0592e1e2-1d07-4c74-990a-08306d4cae06",
+		"name": "Fungus Toast",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Get User Games",
+			"protocolProfileBehavior": {
+				"disableBodyPruning": true
```
Comment (daveshah) [inline FungusToast.postman_collection.json]: I have no idea what this option does but it sounds like something my grandma would appreciate

File: FungusToast.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+						"players",
+						"4",
+						"skills"
+					]
+				}
+			},
+			"response": []
+		}
+	]
+}
```
Comment (daveshah) [inline FungusToast.postman_collection.json]: Digging the idea of checking in the postman collection 👍

File: test/fungus_toast/games/games_test.exs (hunk 1 above)
Comment (daveshah) [reply test/fungus_toast/games/games_test.exs]: Ahh - 🤦‍♂️ good call. I had to view file to see it.

File: test/fungus_toast/games/games_test.exs [hunk 4]
```diff
@@ -142,4 +172,68 @@ defmodule FungusToast.GamesTest do
...
+      game = Games.get_game!(game.id) |> Games.preload_for_games()
+      game.players
+      |> Enum.filter(fn p -> p.human end)
+      |> Enum.map(fn p -> p |> Games.update_player(%{mutation_points: 0}) end)
+
+      game = Games.get_game!(game.id)
+              |> Games.preload_for_games()
+      assert Games.next_round_available?(game)
+    end
+  end
```
Comment (daveshah) [inline test/fungus_toast/games/games_test.exs]: Tests are straightforward and make sense 👍

File: lib/fungus_toast_web/controllers/game_controller.ex [hunk 5]
```diff
@@ -5,18 +5,35 @@ defmodule FungusToastWeb.GameController do
 
   action_fallback FungusToastWeb.FallbackController
 
+  def index(conn, params) do
+    user_id = Map.get(params, "user_id")
+    active = Map.get(params, "active", "false") |> active?()
```
Comment (daveshah) [inline lib/fungus_toast_web/controllers/game_controller.ex]: Kinda of a nitpick, and no real reason to change this, but I think we could delete lines 34-38 if we changed this to 
`active = Map.get(params, "active") in ["1", "true"] ` couldn't we?

Comment (Serneum) [reply lib/fungus_toast_web/controllers/game_controller.ex]: Didn't know that was a thing. We'd no longer return an error, but I think that would be fine

**Pull request merged**